spec/array_rec_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'my_inject_rec'
+
+describe Array do
+  context 'recursive #inject' do
+    it 'takes block to add numbers in an array' do
+      expect([1, 4, 5].inject { |sum, num| sum + num }).to eq 10
+      expect([1, 4, 5].my_inject_rec { |sum, num| sum + num }).to eq 10
+    end
+    it 'takes block to subtract numbers in an array' do
+      expect([1, 4, 5].inject { |sum, num| sum - num }).to eq(-8)
```

Comment:
Lots and lots of use of `[1, 4, 5]` - maybe it would have made sense to spit this out into something like
```ruby
let(:test_arr) { [1, 4, 5] }
```